### PR TITLE
py-bluepysnap: update to v1.0.6

### PR DESCRIPTION
**Pull request by mgeplf**

### CODE REVIEW

File: bluebrain/repo-bluebrain/packages/py-bluepysnap/package.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ class PyBluepysnap(PythonPackage):
     pypi = "bluepysnap/bluepysnap-0.12.0.tar.gz"
 
     version("develop", branch="master")
-    version("1.0.5", sha256="8c9c50bcd7282a96c006c5c7ddffe53542378abc1d400bdf82e91b770d7224e0")
+    version("1.0.6", sha256="78d76354b3076e2e6d5f2332ac8380086854c7890aad86d01d6f21a927f73198")
 
     depends_on("python@3.7:", type=("build", "run"))
```
Comment (GianlucaFicarelli) [inline bluebrain/repo-bluebrain/packages/py-bluepysnap/package.py]: Minor, but we could set python 3.8 as lower limit:
```
    depends_on("python@3.8:", type=("build", "run"))
```

File: bluebrain/repo-bluebrain/packages/py-bluepysnap/package.py [hunk 2]
```diff
@@ -15,7 +15,7 @@ class PyBluepysnap(PythonPackage):
...
     depends_on("py-setuptools", type=("build", "run"))
```
Comment (GianlucaFicarelli) [inline bluebrain/repo-bluebrain/packages/py-bluepysnap/package.py:21]: Minor, but pandas isn't capped in setup.py, so the upper limit could be removed:
```
    depends_on("py-pandas@1.0.0:", type=("build", "run"))
```

File: bluebrain/repo-bluebrain/packages/py-bluepysnap/package.py (hunk 1 above)
Comment (mgeplf) [reply bluebrain/repo-bluebrain/packages/py-bluepysnap/package.py]: tnx

File: bluebrain/repo-bluebrain/packages/py-bluepysnap/package.py (hunk 2 above)
Comment (mgeplf) [reply bluebrain/repo-bluebrain/packages/py-bluepysnap/package.py:21]: tnx

**Review by GianlucaFicarelli — APPROVED** [review]

**Pull request merged**